### PR TITLE
Implement bulk insert capability. issue #16

### PR DESCRIPTION
**Pull request by arcturial**
BulkInsert handler for Common, MySQL, Postgress, SQLserv, SQLite.

### CODE REVIEW

**Comment by harikt** [comment]
:+1:  good work dude .

File: src/Common/BulkInsert.php [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+     * @var array
+     */
+    protected $cols = array();
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function addCols(array $cols)
+    {
+        foreach ($cols as $key => $val) {
```
Comment (stof) [inline src/Common/BulkInsert.php]: `$key` is unused

**Comment by arcturial** [comment]
@stof, removed `$key`

**Comment by pmjones** [comment]
Good on you, @arcturial, for doing the work. I will review and respond over the next few days.

**Comment by pmjones** [comment]
Hi @arcturial -- I began reviewing your PR this morning. It looked fine, but needed some touching up. While I was doing the touching-up work I had a revelation that would allow the BulkInsert to be integrated into the regular Insert. It's a major modification from your PR, but I would not have thought of it if I had not seen your work first. Please take a look at #57 and see if it suits your needs.

Usage is as in the added test in that PR. The order-of-columns is taken from the first added row. You call `addRow()` to add another row, which collects and then resets the insert values. It used named placeholders instead of sequential ones, and renames them for the row number.

**Pull request merged**